OpenRA.Mods.Common/Widgets/Logic/Ingame/IngameMenuLogic.cs

```diff
@@ -88,14 +88,38 @@ public IngameMenuLogic(Widget widget, World world, Action onExit, WorldRenderer
 
 			abortMissionButton.OnClick = () =>
 			{
-				if (world.IsGameOver)
```

Comment:
I'd prefer it if we could keep this check (in spirit). It would probably need to be a bit smarter and only show when the player has not won, ie. something like
```csharp
if (world.LocalPlayer != null && world.LocalPlayer.WinState != WinState.Won)
   // do the prompt thing
else
{
    onQuit();
    return;
}
```

If there has already been discussion about this, I apologize, I missed it then.